### PR TITLE
feat: add all-in-one artifacts download command, per workflow

### PR DESCRIPTION
**Pull request by Pyrrha**
# Description

Propose an all-in-one command to download all artefacts.

# Reasons

#8537

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [x] Break up walls of text by adding paragraph breaks.
- [x] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [x] Keep the title between 20 and 70 characters.
- [x] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [x] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [x] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [x] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

**Comment by rosieyohannan** [comment]
Apologies for the delay with this @Pyrrha , I just need to test it out. WIll do by end of the week 🙇‍♀️

**Comment by Pyrrha** [comment]
@rosieyohannan no worries, I took months from opening the proposal and sending this PR 🙂

File: jekyll/_cci2/artifacts.adoc [hunk 1]
```diff
@@ -168,6 +168,26 @@ Similarly, if you want to download the _latest_ artifacts of a build, replace th
 curl -H "Circle-Token: <circle-token>" https://circleci.com/api/v1.1/project/:vcs-type/:username/:project/latest/artifacts
 ----
 
+In some cases, you may need to download all artefacts from an entire workflow in one time. Having the workflow ID, possibly using the URL, you can use:
```
Comment (rosieyohannan) [inline jekyll/_cci2/artifacts.adoc:171]:
```suggestion
If you need to download all artifacts from a workflow at the same time you can use the following method. You will need your workflow ID.  Copy your workflow ID from the end of the URL when viewing the workflow page in the CircleCI web app:
```

**Review by rosieyohannan — COMMENTED** [review]
Thank you so much for this addition to the arifacts doc! Just one change :-)

**Review by rosieyohannan — APPROVED** [review]

**Pull request merged**